client/src/client.ts

```diff
@@ -15,6 +15,7 @@ import {ProjectLoadingFinish, ProjectLoadingStart, SuggestIvyLanguageService, Su
 import {NgccProgress, NgccProgressToken, NgccProgressType} from '../common/progress';
 import {GetComponentsWithTemplateFile, GetTcbRequest} from '../common/requests';
 
+import {isInsideAngularContext, isInsideInlineTemplateRegion} from './embedded_support';
```

Comment:
nit: I think `isInsideComponentDecorator` is clearer, but I'll let you make the call